projects/canopy/src/assets/icons/play-spot.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M15 28c6.627 0 12-5.373 12-12S21.627 4 15 4 3 9.373 3 16s5.373 12 12 12zm-3-17 8 5-8 5V11z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 500 500"><defs><path id="a" d="M250.09 1.21C112.63 1.21 1.2 112.64 1.2 250.11S112.63 499 250.09 499s248.89-111.43 248.89-248.89S387.55 1.21 250.09 1.21zm-59.94 352.46-.84-207.14 177.72 104.29-176.88 102.85z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>
```

Comment:
The play spot should stay what it was 🤔

Reply:
The play spot needed updating because the alignment was wrong

Reply:
The thing is that the `viewBox` is usually set to `0 0 32 32` 🤔  we need to makes sure the designer that is creating the icons is following the same rules as the other icons and that they update the deign library too.